packages/flutter_tools/lib/src/reporting/custom_dimensions.dart

```diff
@@ -118,8 +116,8 @@ class CustomDimensions {
   final String? commandPackagesAndroidEmbeddingVersion;  // cd46
   final bool? nullSafety;  // cd47
   final bool? fastReassemble;  // cd48
-  final int? nullSafeMigratedLibraries;  // cd49
-  final int? nullSafeTotalLibraries;  // cd50
+  // final int? nullSafeMigratedLibraries;  // cd49
+  // final int? nullSafeTotalLibraries;  // cd50
```

Comment:
Note the deprecation here like you do below.

Reply:
done